save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/service/DockerService.kt

```diff
@@ -194,16 +196,17 @@ class DockerService(
         "UnsafeCallOnNullableType",
         "LongMethod",
     )
-    private fun buildBaseImageForExecution(execution: Execution): Pair<String, String> {
-        val resourcesPath = File(
+    private fun prepareImageAndVolumeForExecution(execution: Execution): RunConfiguration<PersistentVolumeId> {
+        val originalResourcesPath = File(
             configProperties.testResources.basePath,
             execution.resourcesRootPath!!,
         )
-        val agentRunCmd = "./$SAVE_AGENT_EXECUTABLE_NAME"
+        val resourcesForExecution = createTempDirectory(prefix = "save-execution-${execution.id}")
+        originalResourcesPath.copyRecursively(resourcesForExecution.toFile())
 
         // collect standard test suites for docker image, which were selected by user, if any
         val testSuitesForDocker = collectStandardTestSuitesForDocker(execution)
-        val testSuitesDir = resourcesPath.resolve(STANDARD_TEST_SUITE_DIR)
+        val testSuitesDir = resourcesForExecution.resolve(STANDARD_TEST_SUITE_DIR)
```

Comment:
I found some logic with `com.saveourtool.save.utils.StandardTestSuiteUtilsKt#PREFIX_FOR_SUITES_LOCATION_IN_STANDARD_MODE`, will it be affected taking into account that now we copy files to new directory?

Reply:
`DockerServiceKt#getLocationInStandardDirForTestSuite` and `StandardTestSuiteUtilsKt#PREFIX_FOR_SUITES_LOCATION_IN_STANDARD_MODE` are only used to construct a path relative to agent's working dir, so no changes should be required.

This constant is also used in `DockerService#copyTestSuitesToResourcesPath` which is unused after 78cc6164739cc0dfd5b1ef212031526a4c058cbd